src/systemdhook.c

```diff
@@ -262,6 +259,119 @@ static char *get_process_cgroup_subsystem_path(const char *id, int pid, const ch
 	return NULL;
 }
 
+/*
```

Comment:
This sets all the directories found in the list in cgroups.  However I do not handle 'unified' any more as I did in the previous patch.  I'm picking up an AVC on it, should I special case it?  The httpd daemon is coming up just fine without it, but the AVC is a bit concerning.  I have other AVC's in the log which I believe will be tied up with the selinux fix.